GVFS/GVFS/CommandLine/DiagnoseVerb.cs

```diff
@@ -160,10 +168,16 @@ private void RecordVersionInformation()
             this.diagnosticLogFileWriter.WriteLine(information);
         }
 
-        private void CopyAllFiles(string sourceRoot, string targetRoot, string folderName, bool copySubFolders, bool hideErrorsFromStdout = false)
+        private void CopyAllFiles(
+            string sourceRoot, 
+            string targetRoot, 
+            string folderName, 
+            bool copySubFolders, 
+            string targetFolderName = null, 

```

Comment:
If you add `targetFolderName` after `hideErrorsFromStdout` we can ensure that the existing callers are still calling of the parameters in the same order they're listed here.